config/brokers/mt-channel-broker/300-config-mtbroker-delivery.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mt-broker-internal-delivery
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+data:
+  _example: |
+    # Configure the internal channels subscription delivery spec
+    delivery: |
+      retry: 10
```

Comment:
IIUC, I think we should document or warn users what this means.

If I set this retry here `retry: 10` and my trigger has `retry: 10` we're actually retrying at most `10 * 10 = 100` times since each time the channel implementation retries due to the delivery in this ConfigMap, the filter deployment (once implemented) will retry 10 times.

Is my understanding correct?

Reply:
Exactly... Maybe we should use this "internal" configuration just for the ingress channel and then for the other channel we use the `BrokerSpec.Delivery`/`TriggerSpec.Delivery`?

Reply:
The problem with that approach is that retries (in `trigger.spec.delivery.retry` or `broker.spec.delivery.retry`) are ephemeral since the filter deployment might crash while it was retrying and in that case, we're not reaching the specified retries in `trigger.spec.delivery.retry` before giving up.

Reply:
I see your point... I guess it makes sense, so maybe we need to just use this internal map for the first channel (the ingress one) and then use BrokerSpec.Delivery/TriggerSpec.Delivery for the filter channel?